src/lem/multiframe.rs

```diff
@@ -965,6 +965,7 @@ mod tests {
                 let mut w = WitnessCS::<F>::new();
                 let computed_size = compute_witness_size::<F>(&slot_type, &store);
                 allocate_slot(&mut w, &None, 0, slot_type, &store).unwrap();
+                assert_eq!(w.input_assignment(), &[F::ONE]);
```

Comment:
This is helpful. Even better would be for `compute_witness_size` to return both the aux *and* witness sizes. For those slot components that are implemented by `SizedWitness`es, we can get their official report rather than inspecting the assignment and making a concrete comparison — requiring us to depend on details of the input assignment that probably should not be considered here. Non-`SizedWitness` slot components can just report 0 as their `num_inputs()` directly for now.